google/cloud/pubsub/internal/publisher_tracing_connection.cc

```diff
@@ -46,16 +46,16 @@ opentelemetry::nostd::shared_ptr<opentelemetry::trace::Span> StartPublishSpan(
   options.kind = opentelemetry::trace::SpanKind::kProducer;
   auto span =
       internal::GetTracer(internal::CurrentOptions())
-          ->StartSpan(
-              topic + " send",
-              {
-                  {sc::kMessagingSystem, "pubsub"},
-                  {sc::kMessagingDestinationName, topic},
-                  {sc::kMessagingDestinationTemplate, "topic"},
-                  {"messaging.pubsub.ordering_key", m.ordering_key()},
-                  {"messaging.message.total_size_bytes", MessageSize(m)},
-              },
-              options);
+          ->StartSpan(topic + " send",
+                      {
+                          {sc::kMessagingSystem, "pubsub"},
+                          {sc::kMessagingDestinationName, topic},
+                          {sc::kMessagingDestinationTemplate, "topic"},
+                          {"messaging.pubsub.ordering_key", m.ordering_key()},
+                          {"messaging.message.total_size_bytes",
+                           static_cast<int>(MessageSize(m))},
```

Comment:
Googling says pubsub message size is 10MB, which would be greater than the max int32

I think we should use `std::int64_t`

Reply:
Done